Source/WebKit/WebProcess/WebPage/RemoteLayerTree/PlatformCALayerRemote.h

```diff
@@ -120,6 +120,8 @@ class PlatformCALayerRemote : public WebCore::PlatformCALayer, public CanMakeWea
     void setSublayerTransform(const WebCore::TransformationMatrix&) override;
 
     void setIsBackdropRoot(bool) override;
+    bool backdropRootIsOpaque() const override;
+    void setBackdropRootIsOpaque(bool) override;
```

Comment:
They can, but the file-local style is to use `override`.

Feels a bit weird to me to have just these new functions differ, but I'm not strongly attached to that opinion.

Reply:
Project style is to use `final` whenever it can be used instead of `override`. Ideally we don’t start developing different file-local styles; I try to resist that when I can.